docs/ops/monitoring/checkpoint_monitoring.zh.md

```diff
@@ -60,6 +60,8 @@ Flink 的 Web 界面提供了`选项卡/标签（tab）`来监视作业的 check
 
 Checkpoint 历史记录保存有关最近触发的 checkpoint 的统计信息，包括当前正在进行的 checkpoint。
 
+注意，对于失败的 checkpoint，指标会被尽最大努力进行更新并且可能不准确。
```

Comment:
```suggestion
注意，对于失败的 checkpoint，指标会尽最大努力进行更新，但是结果可能不准确。
```

Reply:
Nice, I think your translation is better. Can you @wuchong take a look？THX!

Reply:
Yes. I think so.